tests/utils.py

```diff
@@ -0,0 +1,19 @@
+def get_parametrize_markers_args(node):
+    mark_name = 'parametrize'
+    try:
+        return get_markers_args_using_iter_markers(node, mark_name)
+    except AttributeError:
+        return get_markers_args_using_mark_objects(node, mark_name)
+
+
+def get_markers_args_using_iter_markers(node, mark_name):
```

Comment:
I see now that for the tests you actually used node.iter_markers. Let's then also use it in pytest_bdd/utils.py then

Reply:
Done